code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -620,25 +621,28 @@ Code:
 			active1 = find_record("id", href_list["target"], data_core.general)
 			if(active1)
 				active3 = find_record("id", href_list["target"], data_core.security)
-			loc:mode = 451
+			pda.mode = 451
 			mode = 451
 			if(!active3)
 				active1 = null
 
 		if("Send Signal")
 			spawn( 0 )
-				radio:send_signal("ACTIVATE")
+				var/obj/item/radio/integrated/signal/S
+				S.send_signal("ACTIVATE")

```

Comment:
Why are you doing all this S stuff and creating a new var instead of just replacing `radio:send_signal` with `radio.send_signal`, it looks like that should work since radio is already typecasted properly
(this applies to the below ones as well)